plugins/notion/src/notion.ts

```diff
@@ -396,6 +396,7 @@ export function richTextToPlainText(richText: RichTextItemResponse[]) {
 
 export function getFieldDataEntryInput(
```

Comment:
Nit: Since we have `fieldType`, then maybe also check that `property.type` is compatible with it? So, at the beginning of every case check if `fieldType` is good, and return `undefined` if not? And then every `type: ...` can become `type: fieldType` (or rename `fieldType` to `type` and then it's just `{ type, ... }` 😛 ).